scripts/tests/cirque_tests.sh

```diff
@@ -105,11 +97,9 @@ function cirquetest_run_test() {
 
     # After test finished, the container is perserved and networks will not be deleted
     # This is useful when running tests on local workstation, but not for CI.
-    if [ "x$CLEANUP_DOCKER_FOR_CI" = "x1" ]; then
+    if [[ "x$CLEANUP_DOCKER_FOR_CI" = "x1" ]]; then
```

Comment:
There's no obvious benefit from using `[[` over `[` here, right?  Why do it?

Reply:
[[ stays in the current shell, [ runs a new process

Reply:
Sure, but is that a big deal here?  In any case, I'm ok with this; it just seemed a little odd.

Reply:
with everything, I'm trying to encourage best practices.  Do you think this is noise?

`[` is inferior to `[[` in almost every way, so I use it consistently.  the only place `[` is useful (IMHO) is testing whether a variable can be interpreted by the shell as a number

Reply:
I guess from a philosophical point of view my approach is somehow "don't use non-portable stuff unless it brings concrete benefits, in case you end up having to port after all", but it's a pretty weak preference.  Given we're pretty committed to bash here, this does make sense as a best practice.